cinema/models.py

```diff
@@ -35,12 +39,20 @@
         return f"{self.first_name} {self.last_name}"
 
 
+def convert_uploded_image_name(instance: "Movie", filename: str) -> str:
+    filename = f"{slugify(instance.title)}-{uuid4()}" + Path(filename).suffix
+    return Path("upload/movie_img") / Path(filename)
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(
+        null=True, blank=True, upload_to=convert_uploded_image_name
```

Comment:
Ensure that the function `convert_uploaded_image_name` is correctly referenced here after correcting the typo in its name.